app/cdap/components/GridTable/index.tsx

```diff
@@ -51,6 +53,17 @@ export default function GridTable() {
       count: '0',
     },
   ]);
+  const [toaster, setToaster] = useState<ISnackbarToast>({
+    open: false,
+    isSuccess: false,
+  });
+  const handleSnackbar = () => {
+    setToaster({
+      open: true,
+      isSuccess: true,
+      message: `${T.translate('features.WranglerNewUI.Snackbar.labels.datasetSuccess')}`,
```

Comment:
use a const.PREFIX for the path of translation that can be reused in this component